### PR TITLE
doc: add minutes for meeting 4 Jan 2023

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Review by RafaelGSS — APPROVED** [review]

File: meetings/2023-01-04.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  * Geoffrey, don’t think that’s a good enough reason not to inform users that they are using an
+     experimental feature.
+  * discussion ongoing
+
+* doc: change fetch from experimental to stable [#45684](https://github.com/nodejs/node/pull/45684)
+  * Antoine - blocked because webstreams are still experimental
+  * Robert
+    * webstreams are tricky as we don’t have too many people working on them actively
+    * Rafael only problem is performance issues, but should not be a blocker for making
+       fetch experimental
```
Comment (RafaelGSS) [inline meetings/2023-01-04.md:103]:
```suggestion
       fetch stable
```

**Review by RafaelGSS — APPROVED** [review]

File: meetings/2023-01-04.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      of flags or warnings.
+    * Once ☝️ lands, I’ll open a follow-up PR adding requirements for each level, such as flags or
+      warnings.
+    * Concurrent with the second PR, we need to review the docs and update all the currently
+      experimental features to distinguish which experimental level they are. I don’t anticipate us
+      adding any flags or warnings to experimental features that currently lack them, but we can
+      consider doing so if we think they might be warranted.
+    * Once the above is complete, this would become the process for new experimental features
+      going forward, that would be considered the “default” process unless we decide to grant
+      exceptions on a case-by-case basis.
```
Comment (GeoffreyBooth) [inline meetings/2023-01-04.md:86]: All of this belongs below the “Proposal: Subdivide “Experimental” Status” section.

File: meetings/2023-01-04.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    * Antoine - dispatching custom events on globalThis
+    * James already possible use vm module to run with globalThis that extends EventTarget
+    * Matteo raised concerns on brittleness of the pattern as well
+  * Comment on issue
+    We discussed in the TSC meeting today, at this point the consensus seems to be that value
+    does not outway the issues/problems unless there is a compelling use case we are not aware
+    of. We understand it could be useful for some cases (like webworkers) case but feel there are
+    workarounds.
+  * Geoffrey
+    * I’ll open a PR on the `node` repo to update
```
Comment (mhdawson) [inline meetings/2023-01-04.md:78]:
```suggestion
    * I’ll open a PR on the `node` repo to update
```

File: meetings/2023-01-04.md (hunk 2 above)
Comment (mhdawson) [reply meetings/2023-01-04.md]: I think fixed.

**Review by GeoffreyBooth — APPROVED** [review]

**Review by gireeshpunathil — APPROVED** [review]

**Review by RaisinTen — APPROVED** [review]

**Review by BethGriggs — APPROVED** [review]

**Review by richardlau — APPROVED** [review]

**Review by tniessen — APPROVED** [review]

**Pull request merged**